### PR TITLE
Remove redundant build failure message - #2264

### PR DESCRIPTION
**Pull request by arturbosch**
Removes redundant `BuildFailureReport`

### CODE REVIEW

**Comment by arturbosch** [comment]
Needs rebase on top of #2269

File: detekt-cli/src/main/kotlin/io/gitlab/arturbosch/detekt/cli/console/BuildFailureReport.kt [hunk 1]
```diff
@@ -1,34 +0,0 @@
...
-
-    override fun render(detektion: Detektion): String? {
-        val amount = detektion.getOrComputeWeightedAmountOfIssues(config)
-        val maxIssues = config.maxIssues()
-        return when {
-            maxIssues.isValidAndSmallerOrEqual(amount) -> {
-                "Build failed with $amount weighted issues (threshold defined was $maxIssues).".red()
-            }
-            amount > 0 && maxIssues != -1 -> {
-                "Build succeeded with $amount weighted issues (threshold defined was $maxIssues).".yellow()
```
Comment (schalkms) [inline detekt-cli/src/main/kotlin/io/gitlab/arturbosch/detekt/cli/console/BuildFailureReport.kt]: Don't we loose the functionality provided by this LOC?

Comment (arturbosch) [reply detekt-cli/src/main/kotlin/io/gitlab/arturbosch/detekt/cli/console/BuildFailureReport.kt]: Yes, however I consider it redundant as the "yellow" finding messages provide the same information.

We can consider this change as testing if our users actually use and need the "issue weighting" feature. We could deprecate and remove this extra complexity later?

Comment (schalkms) [reply detekt-cli/src/main/kotlin/io/gitlab/arturbosch/detekt/cli/console/BuildFailureReport.kt]: Okay

**Pull request merged**